source/features/collapse-duplicated-reviews.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const singleReviewThreadComments = select.all('.review-comment.unminimized-comment:only-child');
+	for (const comment of singleReviewThreadComments) {
+		if (!select.exists(`.review-comment#${comment.id}:not(:only-child)`)) { // The review comment is not a duplicate
+			continue;
+		}
+
+		// Change the <details> element to a link to the original comment
+		const details = comment.closest('details')!;
+		const commentUrl = new URL(location.href);
+		commentUrl.hash = comment.id;
```

Comment:
What happens if the comment appears on an old/hidden review thread? 

Edit: answering myself: GitHub detects the hashchange and loads the hidden thread 👍